src/Popover/Popover.js

```diff
@@ -380,7 +380,7 @@ const Popover = React.createClass({
 
   render() {
     return (
-      <div>
+      <div style={{display: 'none'}}>
```

Comment:
Either way, it was a lot of extra lines for a one-line fix, and an unnecessary abstraction.  We're never going to add more styles, so I vote to leave it, but if it really bother you that much, I'll change it. :smile: